models/bloggerLogin.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * This class contains the grantAccess() function which we use to 
+ * determine if the user has entered valid login credentials
+ *
+ * @author jess
+ */
+class bloggerLogin {
+    
+// COMMENTED OUT CONSTRUCTOR AS DON"T THINK WE ACTUALLY NEED ONE FOR THIS?    
+//    public $username;
+//    public $password;
+//    
+//    public function __construct($username, $password) {
+//        $this->username = $username;
+//        $this->password = $password; 
+//    }
+    
+    const BLOGUSERNAME = 'BLOGGER';
```

Comment:
Should all the constants go in one place?
Would you like to start constants.php? Then you'd need to include that file in this one to use the constants.

Reply:
Yeh possibly, maybe if more and more constants are getting added then we can look at breaking them out into their own file.